README.md

```diff
@@ -114,6 +114,11 @@ Kevin Batdorf
   - WordPress: [@meaganhanes](https://profiles.wordpress.org/meaganhanes)
   - Sponsor: [GitHub Sponsor Program](https://github.com/sponsors/meaganhanes)
 
+[Patrick Lumumba](https://wpcorner.co/author/patrick-l/)
+  - Twitter: [@lumiblog](https://twitter.com/lumiblog)
+  - WordPress: [@pedromendonca](https://profiles.wordpress.org/lumiblog/)
```

Comment:
looks like the handle should be `@lumiblog` not `@pedromendonca`

Reply:
Yes. It is @lumiblog. 

Thanks, @jeffpaul